specification/workloads/resource-manager/Microsoft.Workloads/connectors/preview/2023-10-01-preview/connectors.json

```diff
@@ -0,0 +1,1098 @@
+          "type": "boolean"
+        }
+      }
+    },
+    "VMBackupData": {
+      "description": "Defines the VM Backup data for a virtual instance for SAP.",
+      "x-ms-discriminator-value": "VM",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BackupData"
+        }
+      ],
+      "properties": {
+        "backupPolicy": {
+          "$ref": "#/definitions/VMBackupPolicyProperties"
+        },
+        "diskExclusionProperties": {
+          "$ref": "#/definitions/DiskExclusionProperties"
+        }
+      },
+      "required": [
+        "backupPolicy"
+      ]
+    },
+    "DBBackupPolicyProperties": {
+      "description": "Defines the policy properties for database backup.",
+      "type": "object",
+      "required": [
+        "name"
+      ],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the DB backup policy."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "./backupTypes.json#/definitions/AzureVmWorkloadProtectionPolicy"
+        }
+      }
+    },
+    "SSLConfiguration": {
+      "description": "Specify the HANA database TLS/SSL properties which will be used for enabling Azure Backup for this database. You need to specify these details if you have enabled secure communication for your HANA database.",
+      "type": "object",
+      "properties": {
+        "sslKeyStore": {
+          "type": "string",
+          "description": "Specify the name of the keystore file that contains the client's identity (eg. sapsrv.pse). The script will search for the file in the appropriate directory depending on the crypto provider mentioned. If this argument is not provided, it is automatically determined by searching in the configuration files."
+        },
+        "sslTrustStore": {
+          "type": "string",
+          "description": "Specify the name of the trust store file that contains the server’s public certificates (eg. sapsrv.pse). The script will search for the file in the appropriate directory depending on the crypto provider mentioned. If this argument is not provided, it is automatically determined by searching in the configuration files."
+        },
+        "sslHostNameInCertificate": {
+          "type": "string",
+          "description": "Specify the hostname as mentioned in the SSL certificate. If this argument is not provided, it is automatically determined by searching in the SSL certificate."
+        },
+        "sslCryptoProvider": {
+          "type": "string",
+          "description": "Specify the crypto provider being used (commoncrypto/openssl). If this argument is not provided, it is automatically determined by searching in the configuration files.",
+          "enum": [
+            "commoncrypto",
+            "openssl"
+          ],
+          "x-ms-enum": {
+            "name": "sslCryptoProvider",
```

Comment:
>sslCryptoProvider [](http://example.com/codeflow?start=21&length=17)

I wonder if this should be SslCryptoProvider.

Reply:
Since we have followed this naming convention of properties names starting with lower case in all the definitions, keeping it same here as well